src/app/layout.tsx

```diff
@@ -15,8 +15,8 @@ import { Poppins } from 'next/font/google'
 const font = Poppins({ subsets: ['latin'], weight: '400' })
 
```

Comment:
```ts
function age() {
  var dob = new Date(2003, 5, 17)
  var diff_ms = Date.now() - dob.getTime()
  var age_dt = new Date(diff_ms)
  return Math.abs(age_dt.getUTCFullYear() - 1970)
}
```

Reply:
add this to get my age instead of hard coded.

Reply:
Will do!